src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1449,7 +1449,11 @@ void func_80A146CC(EnGo* this, GlobalContext* globalCtx) {
 }
 
 void func_80A14798(EnGo* this, GlobalContext* globalCtx) {
-    s32 sp38[2] = { 0x0000003E, 0x00000F64 };
+    EffectTireMarkInit sp38 = {
```

Comment:
While I understand what this means, gorons aren't really tires... maybe skid marks as a name?

Reply:
Don't really want to name things in my cleanup PRs, would rather wait for the whole file to be documented at once.

Reply:
That's fair 👍